src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -12,6 +12,8 @@
                 "switchToAdvancedMode": "Zum erweiterten Modus wechseln",
                 "advancedModeTooltip": "Der erweiterte Modus bietet erweiterte Konfigurationsmöglichkeiten."
             },
+            "create": "Programmierung erstellen",
+            "import": "Programmierung importieren",
```

Comment:
I think it should contain "exercise" and we should not duplicate translations keys